### ISSUE TITLE
Make R version `4.2.1` the `latest`

### ISSUE THREAD

**Issue by chamaoskurumi** [INWTlab/r-docker#12]
@wahani recently created `4.2.1` branch and upgraded the images (base, batch...). 
However, the `latest` branch is still on `4.2.0`.

I don't know if you had not merged `4.2.1` into `latest` on purpose and I don't want to do the merge without consulting you (I'm afraid of possible problems it creates in production where we use `latest`).

I have already created a PR for merging `4.2.1` into `latest` #11 
Once #11 is merged, I need to change the `r-geos` Dockerfile.

**Comment by wahani** [comment]
Please do not create new PRs in this repo. The branch name is used as a tag and published to docker hub.

Use latest as the dev branch, it is never used in production. New features, like geo image, will then be available with a new r version, when we create a new branch from latest, or have to be merged into older r version branches.

For a fix, like version number: we should always refer to latest image, or specific r version, from the rocker project. If latest is still on 4.2.0 that is my mistake. It should refer to latest from rocker r-ver.

**Comment by wahani** [comment]
Already resolved by @chamaoskurumi
